inspirehep/modules/records/api.py

```diff
@@ -90,6 +91,56 @@ def _create_bucket(self, location=None, storage_class=None):
         )
         return bucket
 
+    def attach_file(self, file_name, file):
+        """Attach the file ``file`` to the record's files properties,
+        updating the record's metadata..
+
+        The file is attached only if there is a key in ``documents`` or
+        ``figures`` which key match ``file_name``.
+        Then. the name of the file is changed adding the record's
```

Comment:
This looks like it was written late in the evening xd, probably needs a bit of rewording